modules/core/src/test/resources/catalogs/landsat8/item-v1.json

```diff
@@ -0,0 +1,200 @@
+{
+  "id" : "LC81530252014153LGN00",
+  "stac_version" : "0.9.0",
+  "stac_extensions" : [
+    "eo",
+    "view",
+    "https://example.com/stac/landsat-extension/1.0/schema.json"
+  ],
+  "type" : "Feature",
+  "geometry" : {
+    "type" : "Polygon",
+    "coordinates" : [
+      [
+        [
+          51.33855,
+          72.27502
+        ],
+        [
+          51.36812,
+          75.70821
+        ],
+        [
+          49.19092,
+          75.67662
+        ],
+        [
+          49.16354,
+          72.3964
+        ],
+        [
+          51.33855,
+          72.27502
+        ]
+      ]
+    ]
+  },
+  "bbox" : [
+    49.16354,
+    72.27502,
+    51.36812,
+    75.67662
+  ],
+  "links" : [
+    {
+      "href" : "../../catalog.json",
+      "rel" : "root",
+      "label:assets" : [
+      ]
+    },
+    {
+      "href" : "../catalog.json",
+      "rel" : "parent",
+      "label:assets" : [
+      ]
+    },
+    {
+      "href" : "LC81530252014153LGN00.json",
+      "rel" : "self",
+      "label:assets" : [
+      ]
+    },
+    {
+      "href" : "https://landsatonaws.com/L8/153/025/LC81530252014153LGN0",
+      "rel" : "alternate",
+      "type" : "text/html",
+      "label:assets" : [
```

Comment:
It looks like `label:assets` should become an `Option[List[_]]`

Reply:
Yeah, this is a known issue -- a holdover from https://github.com/geotrellis/geotrellis-server/issues/152

We treat everything as being part of the label extension :( - I think if we switch it to option then we'll need to change the json printer to exclude `None/null` when printing json.

Reply:
I think this is really a job for the extensions strategy we end up with -- this is a good small example of something where we can't just ship json properties around to solve the problem, since it's at the root of the object that has extension fields